src/Integration.Vsix/SonarLintTagger/TaggerProvider.cs

```diff
@@ -152,6 +153,11 @@ private void OnAnalysisRequested(object sender, AnalysisRequestEventArgs args)
         /// </summary>
         public ITagger<T> CreateTagger<T>(ITextBuffer buffer) where T : ITag
         {
+            if (buffer is IProjectionBuffer)
+            {
+                return null;
+            }
```

Comment:
Could you have made the fix in `ITaggableBufferIndicator` instead rather than repeating the check?
Or would that affect too many taggers?